pkg/controller/build/build_controller.go

```diff
@@ -84,8 +85,11 @@ func (r *ReconcileBuild) Reconcile(request reconcile.Request) (reconcile.Result,
 
 	b := &build.Build{}
 	err := r.client.Get(context.Background(), request.NamespacedName, b)
-	if err != nil {
+	if err != nil && !api_errors.IsNotFound(err) {
```

Comment:
Could you simply return `err` in this case. The error will contain the message informing the resource is not found, and therefore, I think you can simply expose this error.

Reply:
Cool, yes, avoid the duplicated message. Fixed in the second PR:
https://github.com/redhat-developer/build/pull/153/commits/264a9043bb9e56bddcb26f6ff8540a6c9dc8530e

Reply:
In this case as well, maybe you want to cherry-pick the changes on both PRs.

Reply:
wrong word. in the second commit of this PR.

Reply:
Got you. It looks a lot better.